cli/command/service/logs.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/docker/cli/cli/command"
 	"github.com/docker/cli/cli/command/completion"
 	"github.com/docker/cli/cli/command/idresolver"
-	"github.com/docker/cli/service/logs"
+	"github.com/docker/cli/cli/internal/logdetails"
```

Comment:
Still wondering if we need to keep a separate top-level `internal/` and `cli/internal` - not sure if we need to, but we can decide at some point which one makes most sense.